cmd/init.go

```diff
@@ -43,6 +44,112 @@ func (h *Handler) initNew(ctx context.Context, req *entity.CommandRequest) error
 	return nil
 }
 
+func (h *Handler) initFromTemplate(ctx context.Context, req *entity.CommandRequest) error {
+	// Prompt for a template to use
+
+	template, err := ui.PromptStarterTemplates()
+	if err != nil {
+		return err
+	}
+
+	// Select GitHub org
+
+	ui.StartSpinner(&ui.SpinnerCfg{Message: "Fetching GitHub scopes"})
```

Comment:
ultranit: newline for readability/simple extension
```suggestion
	ui.StartSpinner(&ui.SpinnerCfg{
	    Message: "Fetching GitHub scopes",
	})
```